packages/jsii-dotnet-analyzers/src/Amazon.JSII.Analyzers/tools/install.ps1

```diff
@@ -0,0 +1,58 @@
+param($installPath, $toolsPath, $package, $project)
```

Comment:
What's the use of this? How are people supposed to be using this?

Reply:
The only (ugly way) to use an analyzer inside an IDE is to reference its dll directly.

So, in order to avoid that, Microsoft gave Roslyn Analyzers vended by NuGet packages the option to be installed as analyzers, which really just adds this dll reference (not visible in the csproj though).

This is done by adding an install and uninstall scripts in the nupkg. When the analyzer is pulled from NuGet, it gets installed using that script in tools/install.ps1

I have tested this with Rider and Visual Studio. 

See: https://docs.microsoft.com/fr-fr/nuget/guides/analyzers-conventions